src/app/views/orders/order.html

```diff
@@ -36,102 +35,80 @@ <h1 translate>mnoe_admin_panel.dashboard.order.order_info</h1>
         </div>
         <div class="bs-row row">
           <div class="col-xs-6">
-            <div class="label-cell" translate>mnoe_admin_panel.dashboard.order.fullfilment</div>
+            <div class="label-cell" translate>mnoe_admin_panel.dashboard.order.approval</div>
           </div>
-          <div class="col-xs-6">
-            {{vm.order.fullfilment}}
+          <div class="col-xs-3" ng-hide="!vm.displayApproval()">
+            <button role="button" ng-click="vm.approveOrder()" ng-disabled="vm.disableApproval()" class="btn btn-primary btn-xs" translate>
+              mnoe_admin_panel.dashboard.order.approve
+            </button>
           </div>
-          <select class="selectpicker">
-            <option translate>mnoe_admin_panel.dashboard.order.fullfilment_yes</option>
-            <option translate>mnoe_admin_panel.dashboard.order.fullfilment_no</option>
-          </select>
-          <button role="button" ng-click="" class="btn btn-primary btn-xs" translate>
-            mnoe_admin_panel.dashboard.order.save
-          </button>
-        </div>
-        <div class="bs-row row">
-          <div class="col-xs-6">
-            <div class="label-cell" translate>mnoe_admin_panel.dashboard.order.amount</div>
+          <div class="col-xs-3" ng-hide="!vm.displayFulfillApproval()">
+            <button role="button" ng-click="vm.fulfillOrder()" ng-disabled="vm.disableFulfillApproval()" class="btn btn-primary btn-xs" translate>
+              mnoe_admin_panel.dashboard.order.fulfill
+            </button>
           </div>
-          <div class="col-xs-6">
-            {{vm.order.amount}}
+          <div class="col-xs-3">
+            <button role="button" ng-click="vm.cancelOrder()" ng-disabled="vm.disableCancel()" class="btn btn-primary btn-xs" translate>
+              mnoe_admin_panel.dashboard.order.reject
+            </button>
           </div>
         </div>
       </mno-widget-body>
     </mno-widget>
   </div>
   <div class="col-xs-6">
-    <mno-widget is-loading="" icon="fa-user-o" heading="{{vm.order.customer.email}}">
+    <mno-widget is-loading="" icon="fa-user-o" heading="{{'mnoe_admin_panel.dashboard.order.customer' | translate}}">
       <mno-widget-header></mno-widget-header>
       <mno-widget-body>
         <div class="bs-row row">
           <div class="col-xs-4">
             <div class="label-cell" translate>mnoe_admin_panel.dashboard.order.name</div>
           </div>
           <div class="col-xs-6">
-            {{vm.order.customer.name}} {{vm.order.customer.surname}}
+            {{vm.user.name}} {{vm.user.surname}}
           </div>
         </div>
         <div class="bs-row row">
           <div class="col-xs-4">
             <div class="label-cell" translate>mnoe_admin_panel.dashboard.order.email</div>
           </div>
           <div class="col-xs-6">
-            {{vm.order.customer.email}}
+            {{vm.user.email}}
           </div>
         </div>
         <div class="bs-row row">
```

Comment:
`bs-row` can be removed everywhere. It is an old legacy, that has been removed since. Make sure there is a row though.